### DISCUSSION TITLE
create a room in Club House

### DISCUSSION THREAD

**Discussion by varghesejose2020** [EddieHubCommunity/support#1774] [Ideas] [answered]
Hello everyone, This is just a idea to increase the community intraction create room in clubhouse  for community boostup

**Comment by Vyvy-vi** [comment, marked as answer]
That can be a good idea, but right now we use `Discord Stages` for interaction.
This helps us keep track of one platform instead of having to focus on the clubhouse room too

In future, perhaps we could pick a room up 🤔

**Reply by Aub3naad** [reply]
اريد مل مشكله الدخول بعد ادخال رمز التاكيد من خلال الرقم ... اقم وبوضع الكود ولا يتم تحميل حل الدخول

**Comment by eddiejaoude** [comment]
We use TwitterSpaces and Discord Stages. 

I did try club house, it seemed mostly about get rich quick schemes 🤷‍♂️
